hazelcast/src/main/java/com/hazelcast/cache/impl/CacheEventSet.java

```diff
@@ -126,4 +127,13 @@ public int getId() {
     public int getFactoryId() {
         return CacheDataSerializerHook.F_ID;
     }
+
+    @Override
+    public String toString() {
+        return "CacheEventSet{"
+                + "eventType=" + eventType
+                + ", events=" + events
```

Comment:
I really have no idea :) I'm also a bit puzzled since both `CacheEventSet` and `CacheEventData` have the event type, but only `CacheEventData` has the cache name. I'm not sure if there could be `CacheEventData` of different event types in a single set.

For the plugin sample key I'm using the type from the first `CacheEventData` for now (sampling all items will require a refactoring).